### PR TITLE
Refactor Calculator.js to functional component

### PR DESCRIPTION
**Pull request by ShinhyoBelok**
# Changes ♻️ 

* Refactor Calculator.js from class based component to a functional component. 😄

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,64 +1,111 @@
...
+//                 <button type="button" onClick={this.btnClick}>+</button>
+//                 <button type="button" onClick={this.btnClick}>=</button>
+//               </div>
+//             </div>
+//           </div>
+//         </main>
+//       </>
+//     );
+//   }
+// }
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] Kindly remove commented code as they make your code ugly and unreadable.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,64 +1,111 @@
...
+//   }
 
-  btnClick(e) {
-    const state = calculate(this.state, e.target.textContent);
-    this.setState(state);
-  }
+//   btnClick(e) {
+//     const state = calculate(this.state, e.target.textContent);
+//     this.setState(state);
+//   }
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] Kindly remove commented code.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -1,64 +1,111 @@
...
+// class Calculator1 extends React.Component {
+//   constructor(props) {
+//     super(props);
+//     this.state = {
+//       total: 0,
+//       next: null,
+//       operation: null,
+//     };
+//     this.btnClick = this.btnClick.bind(this);
+//   }
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] Remove commented code.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @ShinhyoBelok,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by codecaiine — APPROVED** [review]
Hi @ShinhyoBelok , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- App work as expected :heavy_check_mark: 
- No linters errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**